product-docs/installation/vps.md

```
@@ -0,0 +1,90 @@
+
+
+
+You can go back and update the docker-compose.yml according to your needs or restart the interactive guide to create a new one.
+
+
+
+You can choose Traefik or BunkerWeb instead of Caddy using the config builder. Please note that BunkerWeb deployment is still experimental at the moment.
+
+
+
+### Notes
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)